pkg/agent/cri/cri.go

```diff
@@ -5,10 +5,37 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+	k8sutil "k8s.io/cri-client/pkg/util"
 )
 
 const maxMsgSize = 1024 * 1024 * 16
 
+// Connection connects to a CRI socket at the given path.
+func Connection(ctx context.Context, address string) (*grpc.ClientConn, error) {
```

Comment:
this function was the exact same for both windows and linux, except for use of a socketPrefix and an alias on the util package import.